classes/01class/exercises/c01-aws01/raghunadhpokkalath/README.md

```diff
@@ -0,0 +1,447 @@
+# C01-AWS01
+
+## Commands Execution Output
+
+- Commands to create the first EC2 instance and any additional resource required:
+```
+Create keypair to connect to EC2 instance
+
+> aws ec2 create-key-pair --key-name EC2KeyPair --query 'KeyMaterial' --output text > EC2KeyPair.pem
+
+Vertify fingerprint of keyPair
+
+> aws ec2 describe-key-pairs --key-name EC2KeyPair
+
+{
+    "KeyPairs": [
+        {
+            "KeyPairId": "key-0462d8111f72767a7",
+            "KeyFingerprint": "40:55:75:5c:06:d8:1a:2d:9c:6b:ae:d0:b9:fd:13:db:68:d6:2e:29",
+            "KeyName": "EC2KeyPair",
+            "Tags": []
+        }
+    ]
+}
+
+Create a default vpc (I have deleted default vpc before)
+
+> aws ec2 create-default-vpc
+
+Create security group for EC2
+
+> aws ec2 create-security-group --group-name ec2-sg --description "EC2 Security Group"
+
+{
+    "GroupId": "sg-0c43ee1f951df16a7"
+}
+
+Add CIDR to security group to allow ssh connection
+
+> aws ec2 authorize-security-group-ingress --group-id sg-0c43ee1f951df16a7 --protocol tcp --port 22 --cidr 58.96.95.0/24
```

Comment:
nice that you have added your home public IP here (I am guessing) but using `/24` means that any IP from `58.96.95.1` to `58.96.95.254` can access your instance. You should use something like `58.96.95.XYZ/32`

Reply:
Thanks @caiocezart  fixed this